README.md

```diff
@@ -31,6 +31,9 @@ The CLI must support workflows around troubleshooting and debugging to help when
 * **Deliver applications continuously.**
 While the ECS CLI can be used to manually deploy changes to an application, we always help customers to move to CI/CD by helping them set up and manage pipelines.
 
+#### Security disclosures
+
+If you think you’ve found a potential security issue, please do not post it in the Issues.  Instead, please follow the instructions [here](https://aws.amazon.com/security/vulnerability-reporting/) or email AWS security directly at [aws-security@amazon.com](mailto:aws-security@amazon.com).
```

Comment:
Spacing between sentences is off.

Reply:
How did you catch that :O

Reply:
:man_shrugging: stuck out like a sore :+1:

Reply:
Maaaaaan, is this 30? Not seeing anything anymore?